collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/EntityFormTest.kt

```diff
@@ -21,15 +22,20 @@ class EntityFormTest {
         .around(rule)
 
     @Test
-    fun fillingEntityRegistrationForm_createsEntityInTheBrowser() {
+    fun fillingEntityRegistrationForm_beforeCreatingEntityList_doesNotCreateEntityForFollowUpForms() {
```

Comment:
This title sounds confusing to me similar to https://github.com/getodk/collect/pull/6108#discussion_r1587727889
I'm not sure what `CreatingEntityList` means. First, you create local entities using the registration form and then you open a form that has online entities. So `CreatingEntityList` means opening a form with online entities?

Reply:
You "create an entity list" in the browser using the "Add entity list" flow. Does that make sense or do you think it needs updated?

Reply:
> You "create an entity list" in the browser using the "Add entity list" flow.

But you don't do that here?

Reply:
> But you don't do that here?

Right. That's the "beforeCreatingEntityList" part. I'm testing that we don't create entities if we haven't manually created an entity list yet.